website/blog/2024-07-25-AgentOps/index.mdx

```diff
@@ -0,0 +1,70 @@
+
+The best part is that it only takes two lines of code. All you need to do is set an `AGENTOPS_API_KEY` in your environment (Get API key here: https://app.agentops.ai/account) and call `agentops.init()`:
+```
+import os
+import agentops
+
+agentops.init(os.environ["AGENTOPS_API_KEY"])
+```
+
+## AgentOps' Features
+
+AgentOps includes all the functionality you need to ensure your agents are suitable for real-world, scalable solutions.
+
+<img src="img/dashboard.png" alt="AgentOps overview dashboard" style="max-width: 40%;" />
+
+* **Analytics Dashboard:** The AgentOps Analytics Dashboard allows you to configure and assign agents and automatically track what actions each agent is taking simultaneously. When used with AutoGen, AgentOps is automatically configured for multi-agent compatibility, allowing users to track multiple agents across runs easily. Instead of a terminal-level screen, AgentOps provides a superior user experience with its intuitive interface. 
+* **Tracking LLM Costs:** Cost tracking is natively set up within AgentOps and provides a rolling total. This allows developers to see and track their run costs and accurately predict future costs.
+* **Recursive Thought Detection:** One of the most frustrating aspects of agents is when they get trapped and perform the same task repeatedly for hours on end. AgentOps can identify when agents fall into infinite loops, ensuring efficiency and preventing wasteful computation.
+
+AutoGen users also have access to the following features in AgentOps: 
+
+* **Replay Analytics:** Watch step-by-step agent execution graphs.
+* **Custom Reporting:** Create custom analytics on agent performance.
+* **Public Model Testing:** Test your agents against benchmarks and leaderboards.
+* **Custom Tests:** Run your agents against domain-specific tests.
+* **Compliance and Security:** Create audit logs and detect potential threats, such as profanity and leaks of Personally Identifiable Information.
+* **Prompt Injection Detection:** Identify potential code injection and secret leaks.
+
+## Conclusion 
+
+By integrating AgentOps into AutoGen, we've given our users everything they need to make production-grade agents, improve them, and track their performance to ensure they're doing exactly what you need them to do. Without it, you're operating blindly, unable to tell where your agents are succeeding or failing. AgentOps provides the required observability tools needed to monitor, debug, and optimize your agents for enterprise-level performance. It offers everything developers need to scale their AI solutions, from cost tracking to recursive thought detection. 
+
+Do you find this note helpful? Would you like to share your thoughts, use cases, and findings? Please join our observability channel in the [AutoGen Discord](https://discord.gg/M3kG97qb).
```

Comment:
The discord link didn't work for me? It may be because I'm already on it but can you please double check it.

Reply:
Good catch. The discord link was set to expire after 7 days. I've replaced it with a permanent link